sound/soc/sof/ipc4-topology.c

```diff
@@ -1404,8 +1404,11 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 			struct sof_ipc4_copier_data *alh_data;
 			struct sof_ipc4_copier *alh_copier;
 			struct snd_sof_widget *w;
+			u32 ch_count = 0;
 			u32 ch_mask = 0;
 			u32 ch_map;
+			u32 step;
+			u32 mask;
```

Comment:
nitpick: uninitialized variables can be placed in one line (if I recall right @plbossart prefers this style..)

Reply:
yes I do prefer one variable per line.